src/chrome/content/rules/RaceForward.org.xml

```diff
@@ -0,0 +1,9 @@
+<ruleset name="RaceForward.org">
+	<target host="raceforward.org" />
+	<target host="www.raceforward.org" />
+	<target host="assets.raceforward.org" />
```

Comment:
Do you have a test url for this?

Reply:
No, and I have absolutely no recollection of how I found the subdomain, especially because I wasn't using Sublist3r.

Reply:
Think I might have found it because it was in the SNI for one of the certificates.

Reply:
Okay, it's not a big deal. Let's keep it this way.